docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,128 @@
 
-```rust
-window.listen("tauri://update-available".to_string(), move |msg| {
-  println!("New version available: {:?}", msg);
-})
-```
+`"active"` must be a boolean. By default it's set to false.
 
-#### Javascript
+`"endpoints"` must be an array of updater endpoint URLs as strings. TLS is enforced in production mode.<br/>
+Each updater URL can contain the following variables allowing you to determine [server-side](#update-server-json-format) if an update is available:
 
-```js
-import { listen } from '@tauri-apps/api/event'
-listen('tauri://update-available', function (res) {
-  console.log('New version available: ', res)
-})
-```
+- `{{current_version}}`: The version of the app that is requesting the update.
+- `{{target}}`: The operating system name (one of `linux`, `windows` or `darwin`).
+- `{{arch}}`: The architecture of the machine (one of `x86_64`, `i686`, `aarch64` or `armv7`).
 
-#### Emit Install and Download events
+`"pubkey"` must be a valid public key generated with Tauri's CLI [above](#signing-updates).
 
-You need to emit this event to initialize the download and listen to the [install progress](#listen-install-progress).
+`"dialog"` if present must ba a boolean. By default it's set to true. If enabled, updater [events](#events) will be disabled as the built-in dialog handles everything. If you need custom events, you must turn off the built-in dialog.
 
-Event: `tauri://update-install`
+## Update artifacts
 
-#### Rust
+Tauri's bundler will automatically generate and sign update artifacts once the updater is correctly configured and enabled.
 
-```rust
-window.emit("tauri://update-install".to_string(), None);
-```
+Before building your app, you need to set environment variables for the private key and password:
 
-#### Javascript
+- `TAURI_PRIVATE_KEY`: Path or content of your private key
+- `TAURI_KEY_PASSWORD`: Your private key password (optional)
 
-```js
-import { emit } from '@tauri-apps/api/event'
-emit('tauri://update-install')
-```
-
-#### Listen to the Install Progress
-
-Event: `tauri://update-status`
-
-Emitted data:
-
-```
-status    [ERROR/PENDING/DONE]
-error     String/null
-```
+After that, you can run `tauri build` as usual and Tauri will generate the update bundle and its signature.
 
-PENDING is emitted when the download is started and DONE when the installation is complete. You can then ask to restart the application.
+- **macOS**: On macOS, Tauri will create a `.tar.gz` archive from the application bundle inside the `target/release/bundle/macos/` folder:
 
-ERROR is emitted when there is an error with the updater. We suggest listening to this event even if the dialog is enabled.
+  - `myapp.app` - the standard app bundle.
+  - `myapp.app.tar.gz` - the updater bundle.
+  - `myapp.app.tar.gz.sig` - the signature of the update bundle.
 
-#### Rust
+- **Linux**: On Linux, Tauri will create a `.tar.gz` archive from the AppImage inside the `target/release/bundle/appimage/` folder:
```

Comment:
Generally I've been trying to sort Linux, macOS and Windows alphabetically to get a bit more consistency

Reply:
yeah, same actually. Let's not talk about that i thought M comes before L in the alphabet ever again please 😂